woocommerce/woocommerce
# CYS - Move the `ai/patterns` endpoint to woocommerce admin API

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Part of https://github.com/woocommerce/woocommerce/issues/48150



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1.  In a WooExpress store, go to `wp-admin/admin.php?page=wc-admin&path=%2Fcustomize-store`.
2. Start designing a new store and write the description.
3. Confirm the flow finishes without errors and you see the assembler.
4. Install the `WooCommerce Beta Tester` from this [zip](https://github.com/user-attachments/files/16507435/woocommerce-beta-tester.zip).
5. Go to `wp-admin/tools.php?page=woocommerce-admin-test-helper` and click the `Tools` tabs.
6. Open the network tools, run the `Reset Customize Your Store` command, and check you see a request to `ai/patterns`.



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 
CYS - Move the "ai/patterns" endpoint to woocommerce admin API.
</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
